src/components/button/index.njk

```diff
@@ -87,6 +87,12 @@ The `print` parameter can be set to `true` to render a print button. This will t
     patternlibExample({"path": "components/button/examples/button-print/index.njk"})
 }}
 
+### Exit
```

Comment:
Probably don't need this example, but just add the exit param to the ghost button example and mention it there.